cmd/ketch/app_deploy.go

```diff
@@ -125,10 +137,53 @@ func appDeploy(ctx context.Context, cfg config, getImageConfigFile getImageConfi
 		deploymentSpec,
 	}
 	app.Spec.DeploymentsCount += 1
+
+	var watcher watch.Interface
+	if options.wait {
+		watcher, err = watchReconcileEvent(ctx, cfg.KubernetesClient(), &app)
+		if err != nil {
+			return err
+		}
+	}
+
 	if err = cfg.Client().Update(ctx, &app); err != nil {
 		return fmt.Errorf("failed to update the app: %v", err)
 	}
-	fmt.Fprintln(out, "Successfully deployed!")
+
+	// Await for reconcile result
+	if options.wait {
+		maxExecTime := time.NewTimer(time.Second * 20)
```

Comment:
why do we need it?

Reply:
to exit from the function if we didn't get the even